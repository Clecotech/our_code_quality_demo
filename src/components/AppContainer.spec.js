import React from "react";
import { shallow } from "enzyme";
import { AppContainer } from "./AppContainer";

describe("Container", () => {
  it("renders a default posts page", () => {
    const wrapper = shallow(<AppContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
