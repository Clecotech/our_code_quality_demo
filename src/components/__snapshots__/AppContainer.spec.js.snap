// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders a default posts page 1`] = `
Array [
  <NewPost
    key="new-post"
    onPostCreated={[Function]}
/>,
  <PostsList
    key="posts-list"
    onPostDeleted={[Function]}
    posts={
        Array [
            Object {
              "body": "Doe",
              "title": "John",
            },
          ]
    }
/>,
]
`;
